nibabel/filename_parser.py

```diff
@@ -14,6 +14,7 @@
 except NameError:
     basestring = str
 
+from .loadsave import _stringify_path
```

Comment:
We're getting a circular import. Maybe it makes sense to define the method in this file?